### PR TITLE
feat: add elasticsearch jobs consumer

### PR DESCRIPTION
**Pull request by shawnsarwar**

### CODE REVIEW

File: src/aether-elasticsearch-consumer/templates/deployment.yaml [hunk 1]
```diff
@@ -20,7 +20,7 @@ spec:
         release: {{ .Release.Name }}
     spec:
       containers:
-        - name: consumer
+        - name: es-consumer
```
Comment (shawnsarwar) [inline src/aether-elasticsearch-consumer/templates/deployment.yaml]: @MrPink Can we do this here? If so, does it change how it's referenced in the `gateway-manager/services.yml`?

File: src/aether-elasticsearch-consumer/values.yaml [hunk 2]
```diff
@@ -4,13 +4,18 @@ debug: false
...
 app:
   esConsumerConfigPath: "/code/conf/consumer/consumer.json"
   esConsumerKafkaConfigPath: "/code/conf/consumer/kafka.json"
+  tenancyHeader: "x-oauth-realm"
+  consumerName: "ESC"
+  stackDriverLogging: "True"
+  logLevel: "INFO"
   command: "start"
-  port: "9009"
+  port: 9098
```
Comment (shawnsarwar) [inline src/aether-elasticsearch-consumer/values.yaml]: Can this be an int?

File: src/aether-elasticsearch-consumer/values.yaml [hunk 3]
```diff
@@ -22,3 +27,8 @@ app:
   elasticsearch:
     url: "http://127.0.0.1:9200"
     user: admin
+
+  redis:
+    db: 1
+    host: "http://127.0.0.1"
```
Comment (shawnsarwar) [inline src/aether-elasticsearch-consumer/values.yaml]: Do we need to do something else to allow this container to hit redis?

File: src/aether-elasticsearch-consumer/values.yaml [hunk 4]
```diff
@@ -22,3 +27,8 @@ app:
   elasticsearch:
     url: "http://127.0.0.1:9200"
     user: admin
+
+  redis:
+    db: 1
```
Comment (shawnsarwar) [inline src/aether-elasticsearch-consumer/values.yaml]: I don't know how you want to manage database #s for Redis.

File: src/gateway-manager/templates/services.yaml [hunk 5]
```diff
@@ -38,6 +38,28 @@ data:
       ]
     }
 
+  es-consumer.json: |
+    {
+      "name": "es-consumer",
+      "host": "{{ .Values.gw.aether-elasticsearch-consumer.internalService }}",
```
Comment (shawnsarwar) [inline src/gateway-manager/templates/services.yaml]: Please check this reference.

File: src/aether-elasticsearch-consumer/values.yaml (hunk 4 above)
Comment (MrPink) [reply src/aether-elasticsearch-consumer/values.yaml]: did you want to change this to `0` to keep it default

**Pull request merged**